frontend/src/hex/MinimapView.svelte

```diff
@@ -0,0 +1,47 @@
+<style>
+  .minimap {
+    width: 64px;
+    max-width: 64px;
+    height: 75vh;
```

Comment:
```suggestion
    height: 75vh;
```

Why should it only be 75% of the view height?